docs/_posts/andrei9825/2023-06-07-NER__DemographicExtended_Clinical_en.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: model
+title: Demographic_Extended
+author: John Snow Labs
+name: NER__DemographicExtended_Clinical
+date: 2023-06-07
+tags: [en, licensed]
+task: Named Entity Recognition
+language: en
+edition: Healthcare NLP 4.3.1
```

Comment:
you can upload with 4.4.3

Reply:
I can only select the 4.4.2 from the dropwdown